packages/miniflare/test/index.spec.ts

```diff
@@ -2995,6 +2995,46 @@ test("Miniflare: CF-Connecting-IP is preserved when present", async (t) => {
 	t.deepEqual(await ip.text(), "128.0.0.1");
 });
 
+// regression test for https://github.com/cloudflare/workers-sdk/issues/7924
```

Comment:
It might be helpful to a future reader to have a comment explaining the setup of the tests here. E.g.

> The "server" service just returns the value of the CF-Connecting-IP header which would normally be added by Miniflare. If you send a request to with no such header, Miniflare will add one.
> The "client" service makes an outbound request with a fake CF-Connecting-IP header to the "server" service. If the outbound stripping happens then this header will not make it to the "server" service so its response will contain the header added by Miniflare. If the stripping is turned off then the response from the "server" service will contain the fake header.